pgadmin/README.md

```diff
@@ -0,0 +1,31 @@
+# PgAdmin 4 on GOV.UK PaaS
+This quick start/incubator project can be used to deploy [PgAdmin 4](https://www.pgadmin.org/)
+on [GOV.UK PaaS](https://www.cloud.service.gov.uk/). It will detect any bound Postgres databases,
+and connect to them automatically.
+
+
+## Getting started
+Clone this repository, and push it to your target space with the following password.
+Org and space are extracted from the current target to create a unique hostname for the app.
+
+The resulting hostname will be in the form `org-space-pgadmin`.
+
+```
+cf push \
+  --hostname "$(cf target | grep org | awk '{print $2}')-$(cf target | grep space | awk '{print $2}')-pgadmin" \
```

Comment:
You could ⛳️  this a bit more:

```
--hostname "$(cf target | awk '/org/ {print $2}')-$(cf target | awk '/space/ {print $2}')-pgadmin"
```

Reply:
Yeah, I agree :\ I wish `((space))` and `((org))` were automatically interpolatable by CF :(

Reply:
🙅‍♂  I think if we're documenting this for end users, it's best to avoid assuming they've got `jq` installed.

Even `awk` is a bit dubious, but I think it's acceptable.

Maybe it would be better just to say:

```
--hostname "$YOUR_ORG-$YOUR_SPACE-pgadmin"
```

Reply:
This might be clearer and less dependent on bash magic

Reply:
We can avoid using the `--hostname` flag and put this inside the manifest

```
routes:
  - route: ((org))-((space))-pgadmin.london.cloudapps.digital
```

Reply:
I originally had that, @tlwr, but I wanted to make the manifest good-to-go for folks in either region, so I didn't hard-code the domain.

That said, of all of these, I like Toby's suggestion the most. It forces users of this to have a host name which is unique to their org and space by default, rather than letting somebody steal the `pgadmin.` route for themselves by accident.

Reply:
I have resolved to roll with Toby's suggestion.